src/main/java/no/nav/pto/veilarbportefolje/vedtakstotte/KafkaVedtakStatusEndring.java

```diff
@@ -42,6 +43,8 @@
     LocalDateTime timestamp;
     Innsatsgruppe innsatsgruppe;
     Hovedmal hovedmal;
+    Optional<String> veilederIdent;
```

Comment:
@pederpus @stianStensli is this ok...since we dont have always ansvarlig veileder

Reply:
If it works when serializing from json then it is good 👍  There should be a test for it